lib/gruf/errors/debug_info.rb

```diff
@@ -29,8 +29,10 @@ class DebugInfo
       # @param [Array<String>] stack_trace The stack trace generated by the exception as an array of strings
       #
       def initialize(detail, stack_trace = [])
+        stack_trace_limit = Gruf.backtrace_limit.to_i
+        stack_trace_limit = 10 if stack_trace_limit < 0
```

Comment:
You probably want to allow for a `0` limit, which would mean to not limit at all.